apis/vshn/v1/dbaas_vshn_redis.go

```diff
@@ -193,6 +193,10 @@ func (redis *VSHNRedis) GetInstanceNamespace() string {
 	// GetMaintenanceDayOfWeek returns the currently set day of week
 }
 
+func (redis *VSHNRedis) SetInstanceNamespaceStatus() {
+	redis.Status.InstanceNamespace = redis.GetInstanceNamespace()
+}
+
```

Comment:
maybe:
func (v *VSHNRedis) SetInstanceNamespaceStatus() {
	v.Status.InstanceNamespace = v.GetInstanceNamespace()
}

to keep it the same for all services?

Reply:
The functions right below and above this one both use `redis *VSHNRedis`

Reply:
fine for me ^^